internal/util/features.go

```diff
@@ -0,0 +1,70 @@
+/*
+ Copyright 2017 - 2022 Crunchy Data Solutions, Inc.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package util
+
+import (
+	"fmt"
+
+	"k8s.io/apiserver/pkg/util/feature"
+	"k8s.io/component-base/featuregate"
+)
+
+const (
+// Every feature gate should add method here following this template:
+//
+// // beta: v1.X
+// // Enables FeatureName
+// FeatureName featuregate.Feature = "FeatureName"
+//
+// - https://releases.k8s.io/v1.20.0/pkg/features/kube_features.go#L27
+//
+// Feature gates should be listed in alphabetical, case-sensitive
+// (upper before any lower case character) order.
+//
+// TODO(tjmoore4): Features will start here
```

Comment:
:memo: This `TODO` and the one below will go away once the first feature-gated feature is committed.

Reply:
It may still be worth adding to what's in the comments, but it's worth noting I'm working the initial follow-on stories as well, so I'll be adding real examples in the near feature. I am definitely happy to demo as well.

Reply:
🌱 There's a PR upstream to rearrange their similar file: https://pr.k8s.io/109865